### DISCUSSION TITLE
pick up

### DISCUSSION THREAD

**Discussion by zwluoqi** [pixiv/vroid-sdk-developers#59] [Q&A]
Why can my APP only display so few models, while other apps can display more models when I log in with my account。

![image](https://user-images.githubusercontent.com/3363485/204731467-e7a6fd1e-4d20-4bee-b594-6e1f124fc8a6.png)

**Comment by nyamadan** [comment]
Your application may not have passed our review yet.

Please see [this document](https://vroid.pixiv.help/hc/en-us/articles/900000213643-VRoid-SDK-Guidelines#:~:text=After%20passing%20this%20reviewing%2C%20the%20application%20is%20published%20on%20VRoid%20Hub%2C%20which%20will%20increase%20its%20exposure%20and%20enable%20the%20use%20of%20special%20models%20such%20as%20the%20following.).

**Reply by zwluoqi** [reply]
i had passed your review at 24/12/2022。but now it only show these
![image](https://user-images.githubusercontent.com/3363485/212520520-59bd4280-10c3-48a1-882c-9417d5f25a32.png)
 there is my app，could you help me？

https://hub.vroid.com/apps/pKB0P7vPWWwubraI_ej0JjjKaBWcrqAlUHQDCcvFpjo

**Reply by nyamadan** [reply]
[It may be filtered by application settings.](https://vroid.pixiv.help/hc/en-us/articles/900000277086-Application-settings#:~:text=When%20your%20application%20retrieves%20the%20list%20of%20models%2C%20it%20will%20automatically%20filter%20them%20based%20on%20these%20settings%20and%20on%20the%20usage%20conditions%20that%20were%20set%C2%A0for%20the%20models%20(see%20the%20image%20below).)
